### ISSUE TITLE
create_replication source format

### ISSUE THREAD

**Issue by jlstack** [cloudant/python-cloudant#94]
I had little success using create_replication as it is currently set up. I noticed that the format of the source data was different than the format used by the web UI. The current create_replication used the following format for source:

  "source": {
   "url": "",
   "headers": {
    "Authorization": ""
   }

while the web UI used:

"source": "" (with the appropriate url here)

I made the changes to my code to match the ui and the request succeeded. Is there any known reason as to why the current format would not work?

**Comment by alfinkel** [comment]
I'm not sure that I follow exactly what you mean.  I am assuming that your issue is regarding the replication document generated by the `Replicator.create_replication(...)`.  The format of the "source" element and the target element for that matter that we use allows us to pass credentials using the request header with basic authentication so that the userid and password are not being passed in plain text as part of the URL.  I'm not sure what you mean by web UI but when issuing a replication request in the Cloudant dashboard a similar document is generated in the `_replicator` database as is generated by  `Replicator.create_replication(...)`.  I will say that one limitation we currently have is that CouchDB Admin Party mode does not currently work with this library.  We have a separate issue already open for that.  But I don't think that is your problem since you should have seen this well before starting replication.

Take a look at our [create replication](https://github.com/cloudant/python-cloudant/blob/master/tests/unit/db/replicator_tests.py#L100) test that runs as part of the Travis CI and see if clears things up for you w.r.t. how the Replicator is commonly used.  If it does not, then can you provide a code sample of what exactly is not working for you and I'm sure that we can get you squared away.
